### PR TITLE
improve info about chainspecs on landing page

### PR DESCRIPTION
**Pull request by bernardoaraujor**
improve info about chainspecs on landing page
and fix missing `}` on code samples

### CODE REVIEW

File: projects/landing-page/src/App.tsx [hunk 1]
```diff
@@ -129,9 +129,16 @@ const App: React.FunctionComponent = () => {
...
               </CardNetwork>
             </Grid>
-            <SectionRef href="https://github.com/paritytech/substrate-connect/tree/13bd26a1ca2904f8e0b5d04dfa35e82364d37d99/packages/connect/assets">
+            <SectionRef href="https://github.com/paritytech/substrate-connect/tree/main/packages/connect/src/connector/specs">
               Github repo with chainspecs
             </SectionRef>
+            <SectionText>
+            You have to provide a well-known chain name (polkadot, ksmcc3, westend2, rococo_v2_2).
+            Note that these are the "real" names of the chains rather than the
+            intuitive names (e.g. "kusama" or "rococo"). For example, "ksmcc3" is the name
```
Comment (tomaka) [inline projects/landing-page/src/App.tsx:138]:
```suggestion
            names they are more commonly known as (such as Kusama or Rococo). For example, "ksmcc3" is the name
```

File: projects/landing-page/src/App.tsx [hunk 2]
```diff
@@ -129,9 +129,16 @@ const App: React.FunctionComponent = () => {
...
+            for kusama. This matters for chains which have been hard forked. For example,
```
Comment (tomaka) [inline projects/landing-page/src/App.tsx:139]:
```suggestion
            of Kusama. This matters for chains which have been hard forked. For example,
```

File: projects/landing-page/src/App.tsx [hunk 3]
```diff
@@ -129,9 +129,16 @@ const App: React.FunctionComponent = () => {
...
+            "rococo_v2" and "rococo_v2_2" are two different chains.
```
Comment (tomaka) [inline projects/landing-page/src/App.tsx:140]:
```suggestion
            for kusama. This is important for chains which have been hard forked. For example,
            "rococo_v2" and "rococo_v2_2" are two different chains.
```

File: projects/landing-page/src/App.tsx [hunk 4]
```diff
@@ -156,7 +163,7 @@ const App: React.FunctionComponent = () => {
                   mt={2}
                 >{`await api1.rpc.chain.subscribeNewHeads((lastHeader) => {`}</Box>
                 <Box pl={3}>{`console.log(lastHeader.hash);`}</Box>
-                <Box>{`);`}</Box>
+                <Box>{`});`}</Box>
```
Comment (wirednkod) [inline projects/landing-page/src/App.tsx:166]: The missing } is already fixed

**Review by wirednkod — CHANGES_REQUESTED** [review]

File: projects/landing-page/src/App.tsx [hunk 5]
```diff
@@ -129,9 +129,16 @@ const App: React.FunctionComponent = () => {
                 Cumulus and HRMP
               </CardNetwork>
             </Grid>
-            <SectionRef href="https://github.com/paritytech/substrate-connect/tree/13bd26a1ca2904f8e0b5d04dfa35e82364d37d99/packages/connect/assets">
+            <SectionRef href="https://github.com/paritytech/substrate-connect/tree/main/packages/connect/src/connector/specs">
```
Comment (tomaka) [inline projects/landing-page/src/App.tsx:132]: Why do we have this link in the first place?
This should link to https://paritytech.github.io/substrate-connect/api/enums/connect_src.WellKnownChain.html for example, not to source code. Never link to source code from the documentation.

File: projects/landing-page/src/App.tsx (hunk 4 above)
Comment (bernardoaraujor) [reply projects/landing-page/src/App.tsx]: sorry, merged `main` to my branch.

but the previous fix hasn't landed to https://paritytech.github.io/substrate-connect/ yet.
it still shows like this to me:

Comment (wirednkod) [reply projects/landing-page/src/App.tsx]: That is true. I will deploy soon-ish

**Review by wirednkod — APPROVED** [review]
Besides the `lint` issue that breaks the checks, this looks good to me

**Pull request merged**